src/main/java/org/openRealmOfStars/starMap/randomEvent/BadRandomType.java

```diff
@@ -52,11 +52,6 @@ public enum BadRandomType {
    * must fight against it.
    */
   AGGRESSIVE_WILD_LIFE,
-  /**
-   * Planet climate changes to arid. If it is already arid then it turns
-   * to desert.
-   */
-  CLIMATE_CHANGE,
```

Comment:
This breaks following class:
`Open-Realms-of-Stars/src/main/java/org/openRealmOfStars/gui/mapPanel/PopupPanel.java:[117,14]`